packages/grid/x-data-grid/src/tests/export.DataGrid.test.tsx

```diff
@@ -38,10 +38,6 @@ describe('<DataGrid /> - Export', () => {
 
   describe('component: GridToolbar', () => {
     it('should export with the default csvOptions', async function test() {
-      // Safari 13 doesn't have Blob.text().
```

Comment:
So a bug? We are supposed to support Safari 13 https://mui.com/material-ui/getting-started/supported-platforms/#browser

Reply:
I couldn't find cases where `Blob.text()` is being used.